engine/script/src/script.h

```diff
@@ -121,6 +121,8 @@ namespace dmScript
         void (*DeleteScriptWorld)(HScriptWorld script_world);
         // Called once a frame for the world, dt is local time delta (affected by slo-mo etc)
         void (*UpdateScriptWorld)(HScriptWorld script_world, float dt);
+        // Called once a frame for the world, dt is local time delta (affected by slo-mo etc)
+        void (*FixedUpdateScriptWorld)(HScriptWorld script_world, float dt);
```

Comment:
Currently, there are no extensions using this functionality, but since I can see this being added to the dmSDK later, I think consistency is good.